### DISCUSSION TITLE
Horizontal layout People SharePoint

### DISCUSSION THREAD

**Discussion by WesleySwijghuizenOFIFTY** [microsoft-search/pnp-modern-search#3460] [Q&A]
Hello,

I'm building an People Search (Face Book) for our company.

I've got a question. For the people search i've followed the manual from https://github.com/microsoft-search/pnp-modern-search/blob/develop/docs/scenarios/Create-a-useful-People-Search.md

But i would like to have an horizontal view of the people. Not vertical. I can't find the option for it.

![PnP Search V4](https://github.com/microsoft-search/pnp-modern-search/assets/156892801/a66ca500-ec8a-4330-bb18-563e86c50842)

Is this option possible? Now i've only got the vertical option. I've searched the discussions. But can't find an answer.

Kind regards and thanks in advance!

**Comment by kasperbolarsen** [comment]
I haven't seen a layout like that, the closest might be a two column layout like this one , https://github.com/microsoft-search/pnp-modern-search-layouts/blob/main/Results/Handlebars/PeopleSearch%20Two%20Columns/PeopleSearchTwoColumns.md
( I am not sure that repository has been made public yet)
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/1811b3b5-eb5c-4e70-b629-75908efc46b4)

**Reply by WesleySwijghuizenOFIFTY** [reply]
Thank you i will try that!!
